src/code/qualification/runQualificationWorkflow.m

```diff
@@ -42,45 +42,75 @@ function runQualificationWorkflow(WSettings, ConfigurationPlan, TaskList, Observ
             
             % Plot the results
             plotQualificationTimeProfile(WSettings, j, SimResult, ObservedDataSets, TimeProfile.Plot.Curves, TimeProfile.Plot.Axes, PlotSettings);
-            
+            % Pause option for debugging
+            % pause()
             SectionPath = getSection(ConfigurationPlan.Sections, TimeProfile.SectionId);
-            saveas(gcf,fullfile(SectionPath, ['OutputTimeProfile' num2str(j)]), 'png');
+            saveas(gcf,fullfile(SectionPath, ['OutputTimeProfile' num2str(j)]), 'pdf');
             close;
-            disp(j);
         end
     end
     
     
     % Plot GOFMerged
     % Caution, function not finished, to be completed
     % So far save empty plots
-    if strcmp(TaskList{i}, 'GOFMerged')
+    if strcmp(TaskList{i}, 'GOFMergedPlots')
+        
         for j=1:length(ConfigurationPlan.Plots.GOFMergedPlots)
             
             GOFMerged=ConfigurationPlan.Plots.GOFMergedPlots(j);
             
             % Update plot settings if necessary (to be performed)
             % Currently not handled
-            WSettings = getDefaultWorkflowSettings('Qualification',''); close;
+            WSettings = getDefaultWorkflowSettings('Qualification','default'); close;
             
-            ax = getReportFigure(WSettings,1,1,[],'figureformat','landscape');
+            for k=1:length(GOFMerged)
+                
+                % Update plot settings if necessary (to be performed)
+                % Currently not handled
+                if isfield(GOFMerged(j), 'PlotSettings')
+                    nPlotSettings=GOFMerged(j).PlotSettings;
+                else
+                    nPlotSettings=PlotSettings;
+                end
+                
+                % Get all the groups for one GOF merged plot
+                Groups = GOFMerged.Groups;
+                
+                % Plot the Goodness of fit and output GMFE
+                GOFResults=plotQualificationGOFMerged(WSettings,j,Groups,ObservedDataSets,ConfigurationPlan.SimulationMappings, nPlotSettings);
+                
+                % Pause option for debugging
+                % pause()
+                
+                % Check plot type to perform predictedVsObserved, residualsOverTime or both
+                if ~isempty(strfind(GOFMerged.PlotType, 'residualsOverTime'))
+                    SectionPath = getSection(ConfigurationPlan.Sections, GOFMerged.SectionId);
+                    saveas(gcf,fullfile(SectionPath, ['OutputGOFMergedResiduals' num2str(k)]), 'png');
+                    close(gcf);
+                end
+                if ~isempty(strfind(GOFMerged.PlotType, 'predictedVsObserved'))
+                    SectionPath = getSection(ConfigurationPlan.Sections, GOFMerged.SectionId);

```

Comment:
Looks a lot like line 88 to 90. ALso similar to what we have somewhere else for time profile
Can we create a method that takes a SectionId, sections array and a figure and export it somewhere? 

@Yuri05 Or is this an issue because the gcf is passed as argument ? (re . copy vs ref)